README.md

```diff
@@ -87,9 +87,15 @@ export RUST_LOG=debug  # bash
 set -x RUST_LOG debug  # fish
 ```
 
+You'll need to set up a Postgres instance.  Here is one way:
+```bash
+# create a volume for pg data
+docker volume create tmp_db_data
+docker run --name tmp-postgres -e POSTGRES_PASSWORD=postgres -e POSTGRES_USER=postgres -e POSTGRES_DB=penumbra -p 5432:5432 -v tmp_db_data:/var/lib/postgresql/data -d postgres
+
 Start the Penumbra instance (you probably want to set `RUST_LOG` to `debug`):
 ```
-cargo run --bin pd
+cargo run --bin pd start -d postgres_uri
```

Comment:
`pd` now takes the Postgres URI as a parameter instead of an environment variable -- I think we should try to avoid using any environment variables at all, since they end up acting as a kind of hidden shadow config system.